### PR TITLE
fix the puppet nginx deployment

### PR DESCRIPTION
**Pull request by ehaselwanter**

### CODE REVIEW

File: Modulefile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-name    'hardening/nginx_hardening'
+name    'hardening-nginx_hardening'
```
Comment (chris-rock) [inline Modulefile]: that looks strange? We used the same hardening/* for ssh and os. Any specific reason?

File: manifests/jfryman.pp [hunk 2]
```diff
@@ -68,8 +64,8 @@
   ]
 
   # addhardening parameters
-  file { "${conf_dir}/conf.d/90.hardening.conf":
-    ensure => file,
+  file { '/etc/nginx/conf.d/90.hardening.conf':
```
Comment (chris-rock) [inline manifests/jfryman.pp]: We should not hardwire the nginx path.

Comment (ehaselwanter) [inline manifests/jfryman.pp]: sure. that slipped through

**Comment by chris-rock** [comment]
@ehaselwanter great work

**Pull request merged**